test/spec/controllers/foyer/ressources/types.js

```diff
@@ -176,7 +176,8 @@ describe('Controller: FoyerRessourceTypesCtrl', function() {
 
             // then
             expect(scope.individuVM.ressources[0].tnsActiviteType).toBe('bic');
-            expect(scope.individuVM.ressources[0].montantsMensuels).toBeUndefined();
+            expect(scope.individuVM.ressources[0].montantAnnuel).toBe(0);
```

Comment:
Semantics have changed quite a lot. Have you double-checked everything works as expected before updating the test?